Pages/PendingVolunteersList.razor.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using Karma.Models;
+using Karma.Services;
+using MatBlazor;
+using Microsoft.AspNetCore.Components;
+
+namespace Karma.Pages
+{
+    public partial class PendingVolunteersList
+    {
+        [Inject]
+        public IDBServiceProvider m_DBServiceProvider { get; set; }
+
+        public string FilterValue = "";
+
+        private KarmaContext m_karmaContext = new();
+
+        string CurrentUserId { get; set; }
```

Comment:
this should have private if it is used in cs file and Public if it is used in razor file(I think). Also is this property even used here?

Reply:
Added private. And yes, after changes the property is used.